### ISSUE TITLE
CKEditor: Provide a setting to switch skins.

### ISSUE THREAD

**Issue by bd0bd** [backdrop/backdrop-issues#2729]
Sorry, I can not find the config like it is at Drupal:
admin/config/content/ckeditor/edit/Advanced
admin/config/content/ckeditor/editg

Was going to change skin etc.

**Comment by klonos** [comment]
@bd0bd can you please post screenshots of these admin pages in D7?

I think that in order to change the CKEditor skin in Backdrop, you need to create a custom theme (it can be a subtheme of Basis without any changes) and then add the css for the new CKEditor skin in the theme .info file. If I remember correctly, someone else brought this up either on another issue or during a discussion in Gitter, but I cannot remember exactly.

The current UI allows you to change the buttons used in each text format under Configuration -> Content authoring -> Text editors and formats (`/admin/config/content/formats`). There, you can click the "Configure" button for each text format and set which buttons you want by drag and dropping in the "CKEditor Toolbar": section

![image](https://user-images.githubusercontent.com/2423362/26852864-0766b398-4b53-11e7-9160-57cde34e6a57.png)

In #2631, we have switched the Moono skin to Moono-Lisa that matches better with the Seven admin theme. The ticket originally suggested that we also provided an option to be able to switch themes, but we thought that this was overkill. Lets make this issue here a request to do that.

What other configuration do you think we are missing compared to D7?

**Comment by klonos** [comment]
I'm making this a feature request to allow switching CKEditor skins from the admin UI, but this is also a documentation issue. We need instructions on what needs to be done to change the skin on a theme level.

**Comment by Al-Rozhkov** [comment]
No, you can't change skin with custom theme. You have to build your CKeditor with another skin. Then you have to create module to use your custom build of CKeditor.

**Comment by bd0bd** [comment]
I talk not only about skin change - it is minor thing.
I do not see the useful settings we have in Drupal admin/config/content/ckeditor/edit/Advanced

**Comment by Al-Rozhkov** [comment]
Ckeditor in Backdrop core is not a port of d7 contrib Ckeditor module. So every config option that you need should be discussed separatly.
